core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/LimitedListLineConsumer.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.core.util;
+
+/**
+ * Extension of {@link ListLineConsumer} which consumes limited amount of text {@link #writeLine}.
```

Comment:
Not clear what this class limiting. Characters or lines?

Reply:
It limits size of text which is consuming. 
Will fix class description.